src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -46,51 +51,59 @@ int IStreamsExecutor::Config::GetDefaultNumStreams() {
         return 1;
 }
 
-int IStreamsExecutor::Config::GetHybridNumStreams(const Config& config, const int stream_mode) {
+int IStreamsExecutor::Config::GetHybridNumStreams(std::map<std::string, std::string>& config, const int stream_mode) {
     const int num_phy_cores = getNumberOfCPUCores();
     const int num_big_cores = getNumberOfCPUCores(true);
     const int num_small_cores = num_phy_cores - num_big_cores;
+    int big_core_streams = 0;
+    int small_core_streams = 0;
+    int threads_per_stream_big = 0;
+    int threads_per_stream_small = 0;
 
     if (stream_mode == DEFAULT) {
         // bare minimum of streams (that evenly divides available number of core)
         if (0 == num_big_cores % 4) {
-            config._big_core_streams = std::max(4, num_big_cores / 4);
+            big_core_streams = std::max(4, num_big_cores / 4);
         } else if (0 == num_big_cores % 5) {
-            config._big_core_streams = std::max(5, num_big_cores / 5);
+            big_core_streams = std::max(5, num_big_cores / 5);
         } else if (0 == num_big_cores % 3) {
-            config._big_core_streams = std::max(3, num_big_cores / 3);
+            big_core_streams = std::max(3, num_big_cores / 3);
         } else {  // if user disables some cores say in BIOS, so we got weird #cores which is not easy to divide
-            config._big_core_streams = 1;
+            big_core_streams = 1;
         }
 
-        config._threads_per_stream_big = num_big_cores / config._big_core_streams;
-        config._threads_per_stream_small = config._threads_per_stream_big * 2;
+        threads_per_stream_big = num_big_cores / big_core_streams;
+        threads_per_stream_small = threads_per_stream_big * 2;
         if (num_small_cores == 0) {
-            config._big_core_streams = num_big_cores / config._threads_per_stream_big;
-            config._threads_per_stream_small = 0;
-        } else if (num_small_cores < config._threads_per_stream_small) {
-            config._small_core_streams = 1;
-            config._threads_per_stream_small = num_small_cores;
-            config._threads_per_stream_big = config._threads_per_stream_small / 2;
-            config._big_core_streams = num_big_cores / config._threads_per_stream_big;
+            big_core_streams = num_big_cores / threads_per_stream_big;
+            threads_per_stream_small = 0;
+        } else if (num_small_cores < threads_per_stream_small) {
+            small_core_streams = 1;
+            threads_per_stream_small = num_small_cores;
+            threads_per_stream_big = threads_per_stream_small / 2;
+            big_core_streams = num_big_cores / threads_per_stream_big;
         } else {
-            config._small_core_streams = num_small_cores / config._threads_per_stream_small;
+            small_core_streams = num_small_cores / threads_per_stream_small;
```

Comment:
Can it guarantee that it is divided without remainder? Is below case exist?
```
num_small_cores = 8
threads_per_stream_small = 6
```

Reply:
small_core_streams is 1 in this condition. Small core is not full used.